Source/WebCore/rendering/style/TextSizeAdjustment.h

```diff
@@ -42,7 +42,7 @@ class TextSizeAdjustment {
     constexpr bool isNone() const { return m_value == NoTextSizeAdjustment; }
     constexpr bool isPercentage() const { return m_value >= 0; }
 
-    constexpr bool operator==(const TextSizeAdjustment& anAdjustment) const { return m_value == anAdjustment.m_value; }
+    friend constexpr bool operator==(const TextSizeAdjustment&, const TextSizeAdjustment&) = default;
```

Comment:
Since this is just a `float`, clearly it can just be `TextSizeAdjustment` without the `const&`, but I am unclear on the best rule we should follow about when to do that. I haven’t commented every time I see the possibility of omitting `const&`.